packages/designsystem/plugins/vee-validate.ts

```diff
@@ -1,70 +0,0 @@
-// import { isAfter, isValid } from 'date-fns'
```

Comment:
Can you explain why did you remove this file?

Reply:
ah! Finally you've recreated it…

Reply:
In fact I move the file to be under `src/`. This is for typescript, as the rootDir is `src/`. (check the `tsconfig.json`)

Reply:
For Git it is a deletion and a creation not a move.